### ISSUE TITLE
Region support

### ISSUE THREAD

**Issue by murmelbahn** [custom-cards/nintendo-wishlist-card#1]
Hi,

would it possible to add a region support for the sensor?
I've added it today and was wondering myself why Captain Toad didnt show up as a discounted game. Later I saw it was discounted in the german eshop but not in the US:

German:
https://www.nintendo.de/Spiele/Nintendo-Switch/Captain-Toad-Treasure-Tracker-1348071.html
US:
https://www.nintendo.com/games/detail/captain-toad-treasure-tracker-switch/

So maybe it is possible to add some kind of region support?

**Comment by boralyl** [comment]
I looked briefly but it seems each region uses their own mechanism to feed data.  The US store for example uses a 3rd party service for search which allows filtering by titles on sale.  I had looked at a few other countries and they all use something different or there isn't an obvious URL to look up only titles that are on sale.

I have a warning on the component page, but I should probably add one to the card page too.  I can take a look at the German store in particular to see if there is a way to filter, but ultimately it looks like I'd have to write different code for each region.  And it's not as simple as changing a parameter.

I'll reply back here with what I find when I have some spare time.
